packages/app-store/routing-forms/components/react-awesome-query-builder/widgets.tsx

```diff
@@ -7,9 +8,13 @@ import type {
   ProviderProps,
 } from "react-awesome-query-builder";
 
-import { Button as CalButton, SelectWithValidation as Select, TextField, TextArea } from "@calcom/ui";
+import { Button as CalButton, TextField, TextArea } from "@calcom/ui";
 import { Trash, Plus } from "@calcom/ui/components/icon";
 
+const Select = dynamic(
```

Comment:
`Select` is not used by any system field thus it doesn't make sense to load it as part of initial bundle.